pkg/agent/agent_linux.go

```diff
@@ -18,23 +18,170 @@
 package agent
 
 import (
+	"fmt"
 	"net"
+	"strings"
+	"time"
 
+	"github.com/vishvananda/netlink"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/klog/v2"
+
+	"antrea.io/antrea/pkg/agent/config"
+	"antrea.io/antrea/pkg/agent/interfacestore"
 	"antrea.io/antrea/pkg/agent/util"
+	"antrea.io/antrea/pkg/ovs/ovsconfig"
+	utilip "antrea.io/antrea/pkg/util/ip"
 )
 
 // prepareHostNetwork returns immediately on Linux.
 func (i *Initializer) prepareHostNetwork() error {
 	return nil
 }
 
-// prepareOVSBridge returns immediately on Linux.
+// prepareOVSBridge returns immediately on Linux if enableBridge is false.
 func (i *Initializer) prepareOVSBridge() error {
+	if !i.enableBridge {
```

Comment:
Start to think should we move all these code about bridge/uplink management to a separate file, like under pkg/ovs/bridgemgmt.
@tnqn @wenyingd